### DISCUSSION TITLE
Uno-Check .Net6 what else can be done! to get the standard template working?

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#9134] [Q&A] [answered]
Computer spec:                                     **HP Elite G8, Windows 10 Pro 64bit**
VS Version:                                            **VS2022 17.3.0 preview 2.0,**
Uno-SolutionTemplate version:            **4.3.8.10**
Selected Template:                               **.Net6**

What i have tried:                                  **Uno-check attemt to Fix promt **Y** (No result)
                                                              The whole troubleshoot desctiption including running the powershell v7 script, 
                                                              to erase old .net6 SDK folders.  like below!**

**Troubleshooting
If you run into problems with uno-check, you should generally try the following:

Update the tool to the latest version: dotnet tool update -g uno.check --source https://api.nuget.org/v3/index.json
Run with uno-check --force-dotnet to ensure the workload repair/update/install commands run regardless of if uno-check thinks the workload versions look good
If you have errors still, it may help to run the [Clean-Old-DotNet6-Previews.ps1](https://github.com/unoplatform/uno.check/blob/main/Clean-Old-DotNet6-Previews.ps1) script to remove old SDK Packs, templates, or otherwise old cached preview files that might be causing the problem. Try running uno-check --force-dotnet again after this step.
Finally, if you have problems, run with --verbose flag and capture the output and add it to a new issue.**
**Uno-check attemt to Fix promt again Y (No result)**

Problems:                                              **PM cant complete the, (uno-check --pre --force-dotnet --verbose).
                                                              Solution cant find Android Studio Chipmunk | 2021.2.1 Patch 1.
                                                              Cant connect the android emulator of Visual Sudio eigther.
                                                              Did not connect to my Macbook pro jet.**

 
![Net6UnoCheck](https://user-images.githubusercontent.com/95089394/175768247-400ae30d-6b1a-4c9a-81c1-8f1907b346f1.png)
![UnoCeckFixResult](https://user-images.githubusercontent.com/95089394/175768772-4b4827f0-5c0b-4055-aa88-07e27b1839bb.png)
![cspproj](https://user-images.githubusercontent.com/95089394/175769628-25aa5a4f-bf97-435d-adb7-326df02d83f1.png)
![FWL](https://user-images.githubusercontent.com/95089394/175779836-398388b1-fb75-4c6a-9bb4-bea064592d4f.png)

**Comment by joer33304** [comment, marked as answer]
Did you check the updated steps at https://github.com/unoplatform/uno/discussions/7833 ?

**Reply by Daantje007** [reply]
JOER33304... you are my hero! And JL of cource!! This part woes my Fix:

Try running the following command, in your MyProject.Mobile folder:
dotnet workload install macos ios android maccatalyst (with powershell v7core)

**Comment by Daantje007** [comment]
![image](https://user-images.githubusercontent.com/95089394/175830331-85210a18-4b9c-40da-80ca-179ddfec93b1.png)

**Reply by Daantje007** [reply]
Haha
